### PR TITLE
Sqlite: AUTOINCREMENT wasn't present in the create statement when field is primary key.

### PR DESCRIPTION
**Pull request by jd78**
Given the following Employee:

     var Employee = sequelize.define("Employee", {
             id: {
                 type: dataTypes.INTEGER,
                 autoIncrement: true,
                 primaryKey: true
             },
             name: dataTypes.STRING
    });

When the Sync() is called

Then I expect the following create statement:

     CREATE TABLE IF NOT EXISTS `Employees` (`id` INTEGER PRIMARY KEY AUTOINCREMENT, `name` VARCHAR(255));

But got:

     CREATE TABLE IF NOT EXISTS `Employees` (`id` INTEGER PRIMARY KEY, `name` VARCHAR(255));

### CODE REVIEW

**Comment by mickhansen** [comment]
Wonder how all the tests were passing, they definitely depended on auto incremented keys.
Can you squash your commits to one neat well-described one?

File: test/integration/dialects/sqlite/query-generator.test.js [hunk 1]
```diff
@@ -105,6 +105,10 @@ if (dialect === 'sqlite') {
         {
           arguments: ['myTable', {title: 'VARCHAR(255)', name: 'VARCHAR(255)', otherId: 'INTEGER REFERENCES `otherTable` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION'}],
           expectation: 'CREATE TABLE IF NOT EXISTS `myTable` (`title` VARCHAR(255), `name` VARCHAR(255), `otherId` INTEGER REFERENCES `otherTable` (`id`) ON DELETE CASCADE ON UPDATE NO ACTION);'
+        },
+        {
+          arguments: ['myTable', {id: 'INTEGER PRIMARY KEY AUTOINCREMENT', name: 'VARCHAR(255)'}],
```
Comment (mickhansen) [inline test/integration/dialects/sqlite/query-generator.test.js]: We'd probably want to test an actual attribute with `{type: INTEGER, primaryKey: true, autoIncremement: true}` instead

Comment (jd78) [inline test/integration/dialects/sqlite/query-generator.test.js]: This test is already there and was passing, the issue is during the create statement string builder.

**Comment by mickhansen** [comment]
@jd78 Could you possible do a squash so we get a single neat descriptive commit message?

**Comment by jd78** [comment]
Sorry, I made a mess in this branch. I'll create a new (clean) pull request.

**Pull request closed without merging**